absearch/templates/swagger.tpl

```diff
@@ -0,0 +1,143 @@
+swagger: "2.0"
+host: {{HOST}}
+info:
+  title: ABSearch Server
+  description: lightweight a/b testing tool for search options
+  contact:
+    name: Mike Connor
+    email: mconnor@mozilla.com
+  license:
+    name: APLv2
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: {{VERSION}}
+  x-mozilla-services:
+    homepage: https://github.com/mozilla-services/absearch
+schemes:
+    - {{SCHEME}}
+paths:
+    /:
+      get:
+        operationId: getRoot
+        description: returns a description of the service
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: Description returned
+    /__info__:
+      get:
+        operationId: getInfo
+        description: returns the version info
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: Info returned
+    /__heartbeat__:
+      get:
+        operationId: getHeartbeat
+        description: Performs an heartbeat on the app
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: The hearbeat was successfull
+    /1/{prod}/{ver}/{channel}/{locale}/{territory}/{dist}/{distver}:
+      get:
+        operationId: addUserToCohort
+        description: Assign the user to a cohort and returns its settings
```

Comment:
nit: here you have a capitalized verb, where elsewhere you have a lowercase 3rd person form